### DISCUSSION TITLE
Where to Ask Questions about Neon?

### DISCUSSION THREAD

**Discussion by jdrumgoole** [neondatabase/neon#5229] [Announcements]
Hi,

Please direct all your questions about Neon to https://community.neon.tech/ as we only occasionally review content here. 

Joe Drumgoole
Head of Developer Relations
Neon

**Comment by ChenpiDog** [comment]
Would the community consider setting up a channel on Slack or Discord to discuss Neon-related issues?

**Reply by siegerts** [reply]
Hi @ChenpiDog 👋🏻 , just to follow up here. We now have a Discord server @ https://discord.gg/P5SEVe3C . Hope that this helps!
